modules/ballerina-test/src/test/java/org/ballerinalang/test/statements/breakstatement/BreakStmtTest.java

```diff
@@ -15,14 +15,12 @@
 * specific language governing permissions and limitations
 * under the License.
 */
-package org.ballerinalang.model.statements;
+package org.ballerinalang.test.statements.breakstatement;
 
```

Comment:
Shall we move this to package org.ballerinalang.test.statements.break. Change the resource bal path accordingly.

Reply:
We cannot use *break* in a package path since it is a keyword in java.